docs/ref/templates/builtins.txt

```diff
@@ -1831,7 +1858,20 @@ For example, you can apply ``escape`` to fields when :ttag:`autoescape` is off:
         {{ title|escape }}
     {% endautoescape %}
 
-To escape each element of a sequence, use the :tfilter:`escapeseq` filter.
+.. note::
+
+    As mentioned in the :ttag:`autoescape` section, when filters including
+    ``escape`` are chained together, it can result in unexpected
+    outcomes if preceding filters mark a potentially unsafe string as safe due
+    to the lack of escaping caused by :ttag:`autoescape` being ``off``.
+
+    In such cases, similar to the note in the :tfilter:`safe` filter, chaining
+    ``escape`` would not re-escape strings that have already been marked as
```

Comment:
Do we need a hyphen? (see ce586ed6931092d3a5f06df9031cdeb891793ddb) \cc @smithdc1
```suggestion
    ``escape`` would not reescape strings that have already been marked as
```

Reply:
For `en-us` no hypen is required.

The AP Stylebook (a change in 2019) says to not to hypenate double "e" for "re-" and "pre-" combinations. Only exception would be if the sense was changed. e.g. resign (to quit) or re-sign (to sign again).

Reply:
Note that this change results in a spell check error:

```
WARNING: ref/templates/builtins.txt:1868: : Spell check: reescape:  would not reescape strings that have.
```